src/core/openthread-core-default-config.h

```diff
@@ -66,6 +66,16 @@
 #endif  // OPENTHREAD_CONFIG_DEFAULT_CHANNEL
 
 /**
+ * @def OPENTHREAD_CONFIG_DEFAULT_MAX_TRANSMIT_POWER
+ *
+ * The default IEEE 802.15.4 maximum transmit power (dBm)
+ *
+ */
+#ifndef OPENTHREAD_CONFIG_DEFAULT_MAX_TRANSMIT_POWER
+#define OPENTHREAD_CONFIG_DEFAULT_MAX_TRANSMIT_POWER        127
```

Comment:
I wasn't sure what value to put here.  I was hoping that specifying a value greater than what the radio can support would just cause the radio to rail at its maximum power.  I'm open to using a more reasonable value.  Any suggestions?